hashicorp/vault
# UI - jwt auth config model

I noticed that the UI didn't have the OpenAPI fields for the JWT Auth config, so this adds them (though things like display name currently aren't quite right).

In addition, it wasn't possible to enable and auth method with the 'oidc' alias, so I added that as well.

![Screen Shot 2019-03-14 at 12 31 53 PM](https://user-images.githubusercontent.com/39469/54378894-504f0b00-4656-11e9-9b4d-9a6d0fe938ab.png)

To test: 
1) enable both a jwt and an oidc auth method
2) verify that the additional fields for OIDC support display properly
